nautobot/project-static/js/forms.js

```diff
@@ -613,6 +618,15 @@ function initializeInputs(context) {
     initializeDynamicFilterForm(this_context)
     initializeSelectAllForm(this_context)
     initializeMultiValueChar(this_context)
+
+    $(".modal").each(function() {
```

Comment:
This gets all `.modal` in the entire document right? Should it search only within `this_context` instead?

Reply:
Possibly/probably? But in fact, this should probably live outside of `initializeInputs` because I don't see the use case where we are dynamically creating and registering new modals in the current UI paradigm.